src/Pin.js

```diff
@@ -0,0 +1,16 @@
+var Pin = function (){
+  this.standing = true;
+};
+
+  Pin.prototype.fall = function() {
+    this.standing = false;
+  };
+
+  Pin.prototype.state = function() {
+    if (this.standing === false) {
+      return 'knocked down';
+    } else {
+      return 'standing';
+    }
+  };
+
```

Comment:
Choose one way of representing state, not two things that are basically the same thing